src/odemis/util/test/spot_test.py

```diff
@@ -218,5 +218,83 @@ def test_sanity(self):
                         self.assertAlmostEqual(i, yc + 0.5 * (n - 1))
 
 
+class TestRadialSymmetryCenter(unittest.TestCase):
+    """
+    Unit test class to test the behavior of radial_symmetry_center in
+    odemis.util.spot.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a synthetic dataset of 1000 spot images."""
+        n = 1000
+        shape = (9, 9)
+        refractive_index = 1
+        numerical_aperture = 0.95
+        wavelength = 0.55  # [um]
+        magnification = 40
+        pixel_size = 6.5  # [um]
```

Comment:
Would be nicer if you could write this in m (eg 6.5e-6 m)

Reply:
OK